packages/gatekeeper/src/components/TokenExpiry/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const LOGIN_BUTTON_TEXT = 'Renew session';
+export const SESSION_EXPIRED_TEXT =
+  'Your session has expired. Please click the link below to start a new session';
```

Comment:
Please change to something like "Your session has expired. Please click the link below to log in again."